### DISCUSSION TITLE
What is a "universal cookie"?

### DISCUSSION THREAD

**Discussion by symbiont-matthew-pirocchi** [ItsBenCodes/cookies#338] [Q&A] [answered]
All of the package descriptions say something like "Universal Cookies for _____". But I can't find a description of what a "universal cookie" actually is, or how it differs from a regular cookie, and google doesn't turn up much either.

**Comment by kissu** [comment, marked as answer]
Mainly a cookie that works in any kind of environment: Node, Browser, Edge, Workers etc...
It's `universal` regarding its context, and not an actual specific kind of cookie.
